qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -53,6 +55,7 @@ class Command(misc.MinimalLineEditMixin, misc.CommandLineEdit):
     update_completion = pyqtSignal()
     show_cmd = pyqtSignal()
     hide_cmd = pyqtSignal()
+    update_search = pyqtSignal()
```

Comment:
No need to add a new signal here - just connect `textChanged` to the function directly.